File: src/game/powerups/portal/World1Portal.java
package game.powerups.portal;

import game.models.Coordinates;
import game.ui.panels.game.PitchPanel;

import java.awt.*;

public class World1Portal extends WorldPortal{
    public World1Portal() {
        super(null, 1);
    }

    @Override
    Coordinates getDefaultCoords() {
        return (Coordinates.fromRowAndColumnsToCoordinates(new Dimension(3,4),0,-PitchPanel.GRID_SIZE/2));
    }
}

File: src/game/powerups/portal/World1Portal.kt
package game.powerups.portal

import game.entity.EntityTypes
import game.entity.models.Coordinates
import game.ui.panels.game.PitchPanel
import java.awt.Dimension

class World1Portal() : WorldPortal(null, 1) {
    constructor(id: Long) : this() {
        this.entityInfo.id = id
    }

    override val defaultCoords: Coordinates
        get() = Coordinates.fromRowAndColumnsToCoordinates(Dimension(3, 4), 0, -PitchPanel.GRID_SIZE / 2)

    override val type: EntityTypes
        get() = EntityTypes.World1Portal
}